### ISSUE TITLE
Error "Cannot read property 'includes' of undefined"

### ISSUE THREAD

**Issue by yaplusplus** [tldr-pages/tldr-node-client#283]
## Expected behaviour
It shows a man page

## Actual behaviour
It throws an error `Cannot read property 'includes' of undefined` and fails (see log below). Clean install, right after `npm i -g tldr`.

## Log, debug output
I use zsh:
```
tldr git-stash

TypeError: Cannot read property 'includes' of undefined
    at getShortIndex.then (/usr/local/lib/node_modules/tldr/lib/index.js:24:29)
✔ Page not found. Updating cache...
✔ Creating index...
TypeError: Cannot read property 'includes' of undefined
    at getShortIndex.then (/usr/local/lib/node_modules/tldr/lib/index.js:24:29)
Page not found.
```

## Environment
macOS Catalina 10.15.15
node v10.15.1

**Comment by agnivade** [comment]
Can you mention your tldr version (`tldr --version`) ? I am unable to reproduce this.

Please also show us the output of `echo $LANG`.

And also, try with `LANG=en tldr git-stash`

**Comment by yaplusplus** [comment]
```
tldr --version
3.3.1
```

`echo $LANG` returns an empty line.

`LANG=en tldr git-stash` works perfectly!

**Comment by agnivade** [comment]
Does `LANG= tldr git-stash` cause the failure ?

The code does set `en` by default, so I am surprised why is it failing.

**Comment by yaplusplus** [comment]
No, `LANG= tldr git-stash` works correctly.

**Comment by yaplusplus** [comment]
I did a little debugging and found the source of the problem:
```js
getPage(page) {
    let preferredPlatform = platform.getPreferredPlatformFolder(this.config);
    let preferredLanguage = 'en';
    if (process.env['LANG'] !== '') {
      preferredLanguage = process.env['LANG'];
    }
```
In my case `process.env['LANG']` is `undefined`, so `preferredLanguage` becomes undefined too.

**Comment by agnivade** [comment]
Thank you for the analysis. Yes, that is the problem.
